README.md

```diff
@@ -34,25 +34,16 @@ This project uses Swift Package Manager to pull in the frameworks from [axe-devt
 
 _Note: XCUITests are only supported with a real device from Sauce at this time. This project requires a device on iOS 15.0 or later._
 
-In the top center of the Xcode window, tap on the device drop-down and select your desired device.
+Follow our online guide for setting your project up with [XCUITesting on SauceLabs.](https://docs.deque.com/devtools-mobile/ios-example-sauce-labs-xcui) 
 
-<img src="doc_img/Device1.png" alt="Shows the click area for selecting a device."/>
+To run a test with this project, follow the prerequisites in the link above, then come back here.
```

Comment:
Do we need both of these lines? Could we merge them to be like: 

"To run a test with this project, first follow our guide to set up your project with [XCUITesting on SauceLabs]."